src/main/java/com/extendedclip/papi/expansion/essentials/EssentialsExpansion.java

```diff
@@ -391,6 +391,10 @@ public String onRequest(OfflinePlayer player, @NotNull String identifier) {
                 return user.isTeleportEnabled() ? papiTrue : papiFalse;
             case "is_muted":
                 return user.isMuted() ? papiTrue : papiFalse;
+            case "playtime":
+                long playTime = user.getBase().getStatistic(Statistic.PLAY_ONE_MINUTE);
```

Comment:
This will break on older versions. Needs to handle a fallback to PLAY_ONE_TICK.

Reply:
Is it as simple as just adding:
```
if (playTime == 0) {
    playTime = user.getBase().getStatistic(Statistic.PLAY_ONE_TICK) / 1200;
}
```
?